tests/handlebars-basics/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -51,12 +51,12 @@ singleQuote: true
 <div class="{{hello}} {{world}}"></div>
 
 =====================================output=====================================
-<script id="entry-template" type="text/x-handlebars-template">
-  <div class="entry">
+<script id='entry-template' type='text/x-handlebars-template'>
```

Comment:
Problem what we change default `"` on `'`. We should keep `"` quotes for html like syntax (it is preferable developers).

Reply:
No, no, I don't change the default quote in this PR. In that specific test, the `singleQuote` option is true.